gnomad/utils/vcf.py

```diff
@@ -493,6 +493,31 @@ def make_combo_header_text(
     return header_text
 
 
+def create_label_groups(
+    pops: List[str],
+    sexes: List[str],
```

Comment:
since this is in the common repo now, should we set a default for this field too? possibly to this? https://github.com/broadinstitute/gnomad_methods/blob/master/gnomad/utils/vcf.py#L45 (we should also update this constant to use XX and XY now)